pkg/controllers/feature/feature_controller.go

```diff
@@ -462,6 +466,105 @@ func (r *frReconciler) updateFeatureSetStatus(ctx context.Context, fs *uiapi.Fea
 	return client.IgnoreNotFound(err)
 }
 
+func (r *frReconciler) calculateFeatureSetDependency(ctx context.Context) error {
```

Comment:
We are just not calculating. May be `updateAllFeatureSetDependancies()`

Reply:
sounds good